2023-10-29 16:44:02,231 INFO: HYF DB startup [in /home/marta/HackYourFuture/HYF_DB_example/app/__init__.py:56]
2023-10-29 16:44:07,586 ERROR: Exception on /see_user_list [GET] [in /home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/flask/app.py:825]
Traceback (most recent call last):
  File "/home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/flask/app.py", line 1455, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/flask/app.py", line 869, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/flask/app.py", line 867, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/flask/app.py", line 852, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/home/marta/HackYourFuture/HYF_DB_example/app/main/routes_see_data.py", line 26, in see_user_list
    return render_template("see_data.html", title='See users', data=users.items,
  File "/home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/flask/templating.py", line 152, in render_template
    return _render(app, template, context)
  File "/home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/flask/templating.py", line 133, in _render
    rv = template.render(context)
  File "/home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/jinja2/environment.py", line 1304, in render
    self.environment.handle_exception()
  File "/home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/jinja2/environment.py", line 925, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/home/marta/HackYourFuture/HYF_DB_example/app/templates/see_data.html", line 2, in <module>
    {% import 'bootstrap/wtf.html' as wtf %}
  File "/home/marta/HackYourFuture/HYF_DB_example/app/templates/base.html", line 1, in <module>
    {% extends "bootstrap/base.html" %}
  File "/home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 1, in <module>
    {% block doc -%}
  File "/home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 4, in <module>
    {%- block html %}
  File "/home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 20, in <module>
    {% block body -%}
  File "/home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 23, in <module>
    {% block content -%}
  File "/home/marta/HackYourFuture/HYF_DB_example/app/templates/base.html", line 51, in <module>
    {% block app_content %}{% endblock %}
  File "/home/marta/HackYourFuture/HYF_DB_example/app/templates/see_data.html", line 5, in <module>
    <h1>Hi {{ current_user.username }}!</h1>
  File "/home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/jinja2/environment.py", line 474, in getattr
    return getattr(obj, attribute)
jinja2.exceptions.UndefinedError: 'current_user' is undefined
2023-10-29 16:44:07,731 ERROR: Exception on /see_user_list [GET] [in /home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/flask/app.py:825]
Traceback (most recent call last):
  File "/home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/flask/app.py", line 1455, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/flask/app.py", line 869, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/flask/app.py", line 867, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/flask/app.py", line 852, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/home/marta/HackYourFuture/HYF_DB_example/app/main/routes_see_data.py", line 26, in see_user_list
    return render_template("see_data.html", title='See users', data=users.items,
  File "/home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/flask/templating.py", line 152, in render_template
    return _render(app, template, context)
  File "/home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/flask/templating.py", line 133, in _render
    rv = template.render(context)
  File "/home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/jinja2/environment.py", line 1304, in render
    self.environment.handle_exception()
  File "/home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/jinja2/environment.py", line 925, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/home/marta/HackYourFuture/HYF_DB_example/app/templates/see_data.html", line 2, in <module>
    {% import 'bootstrap/wtf.html' as wtf %}
  File "/home/marta/HackYourFuture/HYF_DB_example/app/templates/base.html", line 1, in <module>
    {% extends "bootstrap/base.html" %}
  File "/home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 1, in <module>
    {% block doc -%}
  File "/home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 4, in <module>
    {%- block html %}
  File "/home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 20, in <module>
    {% block body -%}
  File "/home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 23, in <module>
    {% block content -%}
  File "/home/marta/HackYourFuture/HYF_DB_example/app/templates/base.html", line 51, in <module>
    {% block app_content %}{% endblock %}
  File "/home/marta/HackYourFuture/HYF_DB_example/app/templates/see_data.html", line 5, in <module>
    <h1>Hi {{ current_user.username }}!</h1>
  File "/home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/jinja2/environment.py", line 474, in getattr
    return getattr(obj, attribute)
jinja2.exceptions.UndefinedError: 'current_user' is undefined
2023-10-29 16:44:09,437 ERROR: Exception on /see_user_list [GET] [in /home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/flask/app.py:825]
Traceback (most recent call last):
  File "/home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/flask/app.py", line 1455, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/flask/app.py", line 869, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/flask/app.py", line 867, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/flask/app.py", line 852, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/home/marta/HackYourFuture/HYF_DB_example/app/main/routes_see_data.py", line 26, in see_user_list
    return render_template("see_data.html", title='See users', data=users.items,
  File "/home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/flask/templating.py", line 152, in render_template
    return _render(app, template, context)
  File "/home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/flask/templating.py", line 133, in _render
    rv = template.render(context)
  File "/home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/jinja2/environment.py", line 1304, in render
    self.environment.handle_exception()
  File "/home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/jinja2/environment.py", line 925, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/home/marta/HackYourFuture/HYF_DB_example/app/templates/see_data.html", line 2, in <module>
    {% import 'bootstrap/wtf.html' as wtf %}
  File "/home/marta/HackYourFuture/HYF_DB_example/app/templates/base.html", line 1, in <module>
    {% extends "bootstrap/base.html" %}
  File "/home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 1, in <module>
    {% block doc -%}
  File "/home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 4, in <module>
    {%- block html %}
  File "/home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 20, in <module>
    {% block body -%}
  File "/home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 23, in <module>
    {% block content -%}
  File "/home/marta/HackYourFuture/HYF_DB_example/app/templates/base.html", line 51, in <module>
    {% block app_content %}{% endblock %}
  File "/home/marta/HackYourFuture/HYF_DB_example/app/templates/see_data.html", line 5, in <module>
    <h1>Hi {{ current_user.username }}!</h1>
  File "/home/marta/miniconda3/envs/hyf_flask_example/lib/python3.10/site-packages/jinja2/environment.py", line 474, in getattr
    return getattr(obj, attribute)
jinja2.exceptions.UndefinedError: 'current_user' is undefined
